convert2rhel/actions/post_conversion/hostmetering.py

```diff
@@ -0,0 +1,235 @@
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__metaclass__ = type
+
+
+import logging
+import os
+
+from convert2rhel import actions, systeminfo
+from convert2rhel.pkgmanager import call_yum_cmd
+from convert2rhel.subscription import get_rhsm_facts
+from convert2rhel.systeminfo import system_info
+from convert2rhel.utils import run_subprocess
+
+
+logger = logging.getLogger(__name__)
+
+
+class ConfigureHostMetering(actions.Action):
+    id = "CONFIGURE_HOST_METERING_IF_NEEDED"
+
+    def run(self):
+        """
+        Decide whether to install, enable and start host-metering on the system based on the
+        CONVERT2RHEL_CONFIGURE_HOST_METERING environment variable.
+
+        Behavior can be controlled CONVERT2RHEL_CONFIGURE_HOST_METERING environment variable:
+        - "auto": host-metering will be configured based on the above conditions
+        - env var not present: host-metering will not be configured
+        - "force": forces configuration of host-metering (e.g., even if not running on a hyperscaler)
+        - any other value: Will be ignored and host metering will not be configured.
+
+        :return: True if host-metering is configured successfully, False otherwise (used mainly for testing).
+        :rtype: bool
+        """
+        logger.task("Final: Configure host-metering")
+
+        super(ConfigureHostMetering, self).run()
+
+        self.env_var = os.environ.get("CONVERT2RHEL_CONFIGURE_HOST_METERING", None)
+        if not self._check_env_var_correct():
+            return False
+
+        if system_info.version.major != 7 and self.env_var != "force":
+            logger.info("Skipping host metering configuration. Only supported for RHEL 7.")
+            self.add_message(
+                level="INFO",
+                id="SKIPPED_CONFIGURE_HOST_METERING",
+                title="Skipping host metering configuration.",
+                description="Host metering is supportted only for RHEL 7.",
+            )
+            return False
+
+        self._set_should_configure_metering()
+
+        rhsm_facts = get_rhsm_facts()
+        is_hyperscaler = self.is_running_on_hyperscaler(rhsm_facts)
+        if (not is_hyperscaler or not self.should_configure_metering) and self.env_var != "force":
+            logger.info("Skipping host-metering configuration.")
+            self.add_message(
+                level="INFO",
+                id="SKIPPED_CONFIGURE_HOST_METERING",
+                title="Skipping host metering configuration as not needed.",
```

Comment:
```suggestion
                title="Did not perform host metering configuration as not needed.",
```
We want to avoid the use of skip in our messages to not confuse customers with our "SKIP" level. Reference -https://github.com/oamg/convert2rhel/pull/990/files